### DISCUSSION TITLE
Show chart from dashboard with any date range

### DISCUSSION THREAD

**Discussion by timaschew** [firefly-iii/firefly-iii#6395] [Firefly III] [answered]
The chart on the dashboard can be configured to show it for the current month, 3 months, 6 months, 12 months.
But depending on the current date it's somtimes not so helpful because the range goes into the future.

It would be nice if the chart (not neccessary on the dashboard, but on an extra page) could be configured to show any date range, even more than a year.

**Comment by JC5** [comment, marked as answer]
There are reports you can use with your own time range. Would those help?

**Reply by timaschew** [reply]
I didn't find a report which shows the asset accounts separated, only aggregated as net worth. But I would like to see them separated like on the dashboard.

**Reply by JC5** [reply]
You can show any date range if you select "custom" in the date dropdown.

**Reply by timaschew** [reply]
I know that I can set any date there, but it's the "wrong" chart. I want this chart from the screenshot which shows only on the dashboard nowhere else.

<img width="803" alt="Screenshot 2022-09-01 at 07 10 58" src="https://user-images.githubusercontent.com/110870/187836248-e7a52135-6c78-4d9a-bd1b-cd557d1fe837.png">

**Reply by JC5** [reply]
I know what you mean, but that chart updates with the change in the date range. I just tried it at the demo site.

**Reply by timaschew** [reply]
Can you give me some more details in terms of URLs?

For me the range of the chart on the dashboard can only be configured via the `/preferences` page. But I can't use a custom range, only preconfigured ranges (1, 3, 6, 12 months). (And since the range is fixed, they are going sometimes into future).

On the `/reports` page I can set any custom range, but the chart which is shown on the next page (for example `/reports/default/4,5,1/20210825/20220828` shows different charts, not the one from the dashboard (separated asset accounts).

**Reply by JC5** [reply]
There is a date selector on the dashboard (top right). Click it, and select "custom range".

**Reply by timaschew** [reply]
🙈 Oh that's a dropdown, wow. I didn't expect it, but that's exactly what I'm looking for!
Thank you!
